cli/src/main.rs

```diff
@@ -174,11 +174,18 @@ where
 async fn store(label: &str, attributes: &impl AsAttributes) -> Result<(), Error> {
     let collection = collection().await?;
 
-    print!("Type a secret: ");
-    std::io::stdout()
-        .flush()
-        .map_err(|_| Error::new("Could not flush stdout"))?;
-    let secret = rpassword::read_password().map_err(|_| Error::new("Can't read password"))?;
+    let mut stdin = std::io::stdin().lock();
```

Comment:
The commit mentions lookup but then you are changing the store method. Please update the commit message & also include a brief justification of why this change is necessary

Reply:
Done